USAGE.md

```diff
@@ -122,72 +122,104 @@ aws lambda invoke \
+```
 
-- Example of all Datasets listing request
+Please note that it is important to choose an accurate and stable title. Changing the title is
+complex, time-consuming, risky and lossy. If you need to change the title, choose between
+[changing the dataset title by creating a copy of the latest dataset version](#Changing-the-dataset-title-by-creating-a-copy-of-the-latest-dataset-version)
+and
+[changing the dataset title by renaming and moving the files](#Changing-the-dataset-title-by-renaming-and-moving-the-files)
+below.
 
-  ```console
-  $ aws lambda invoke \
-      --function-name "${ENV}-datasets" \
-      --payload '{"http_method": "GET", "body": {}}' \
-      /dev/stdout
+### All datasets listing request
 
-  {"status_code": 200, "body": [{"created_at": "2021-02-01T13:38:40.776333+0000", "id": "cb8a197e649211eb955843c1de66417d", "title": "Auckland 2020", "updated_at": "2021-02-01T13:39:36.556583+0000"}]}
-  ```
+```console
+$ aws lambda invoke \
+    --function-name "${ENV}-datasets" \
+    --payload '{"http_method": "GET", "body": {}}' \
+    /dev/stdout
 
-- Example of single Dataset listing request
+{"status_code": 200, "body": [{"created_at": "2021-02-01T13:38:40.776333+0000", "id": "cb8a197e649211eb955843c1de66417d", "title": "Auckland_2020", "updated_at": "2021-02-01T13:39:36.556583+0000"}]}
+```
 
-  ```console
-  $ aws lambda invoke \
-      --function-name "${ENV}-datasets" \
-      --payload '{"http_method": "GET", "body": {"id": "cb8a197e649211eb955843c1de66417d"}}' \
-      /dev/stdout
+### Single dataset listing request
 
-  {"status_code": 200, "body": {"created_at": "2021-02-01T13:38:40.776333+0000", "id": "cb8a197e649211eb955843c1de66417d", "title": "Auckland 2020", "updated_at": "2021-02-01T13:39:36.556583+0000"}}
-  ```
+```console
+$ aws lambda invoke \
+    --function-name "${ENV}-datasets" \
+    --payload '{"http_method": "GET", "body": {"id": "cb8a197e649211eb955843c1de66417d"}}' \
+    /dev/stdout
 
-- Example of Dataset delete request
+{"status_code": 200, "body": {"created_at": "2021-02-01T13:38:40.776333+0000", "id": "cb8a197e649211eb955843c1de66417d", "title": "Auckland_2020", "updated_at": "2021-02-01T13:39:36.556583+0000"}}
+```
 
-  ```console
-  $ aws lambda invoke \
-      --function-name "${ENV}-datasets" \
-      --payload '{"http_method": "DELETE", "body": {"id": "cb8a197e649211eb955843c1de66417d"}}' \
-      /dev/stdout
+### Dataset delete request
 
-  {"status_code": 204, "body": {}}
-  ```
+```console
+$ aws lambda invoke \
+    --function-name "${ENV}-datasets" \
+    --payload '{"http_method": "DELETE", "body": {"id": "cb8a197e649211eb955843c1de66417d"}}' \
+    /dev/stdout
+
+{"status_code": 204, "body": {}}
+```
+
+### Changing the dataset title by creating a copy of the latest dataset version
+
+This is the simplest way to change a dataset title:
```

Comment:
Do we need to add in the downside of this. Like losing previous versions of the dataset?

Reply:
I've mentioned further down that asking to remove the existing dataset is an optional step. I don't expect we can just unconditionally get rid of the original dataset.

Reply:
I've documented how the datasets have no explicit connection between them.